.eslintrc.js

```diff
@@ -8,5 +8,7 @@ module.exports = {
     },
     rules: {
         'react/display-name': 'off',
+        'react/no-unknown-property': 'off',
+        'react/no-deprecated': 'off',
```

Comment:
I'm not sure why you've disabled these rules? When I run locally, disabling then doesn't seem to make any difference in the eslint check?